expression/helper.go

```diff
@@ -70,6 +70,12 @@ func getTimeValue(ctx context.Context, v interface{}, tp byte, fsp int) (d types
 		upperX := strings.ToUpper(x)
 		if upperX == CurrentTimestamp {
 			value.Time = types.FromGoTime(defaultTime)
+			if tp == mysql.TypeTimestamp {
+				err := value.ConvertTimeZone(time.Local, ctx.GetSessionVars().GetTimeZone())
```

Comment:
In the description, I got this:
> Default value from "timestamp" session variable should be in UTC timezone, rather than based on session's timezone.
Why use time.Local here? Should we use UTC?